e2e/test/scenarios/question/nulls.cy.spec.js

```diff
@@ -81,15 +76,13 @@ describe("scenarios > question > null", () => {
           },
         ],
       },
-      cardDetails: {
-        size_x: 11,
-        size_y: 6,
-      },
```

Comment:
This didn't make any sense after #29641.
Moved it to lines 84 and 85, and slightly tweaked the card size.